lib/AST/Decl.cpp

```diff
@@ -4154,8 +4154,11 @@ unsigned EnumDecl::getODRHash() {
 RecordDecl::RecordDecl(Kind DK, TagKind TK, const ASTContext &C,
                        DeclContext *DC, SourceLocation StartLoc,
                        SourceLocation IdLoc, IdentifierInfo *Id,
-                       RecordDecl *PrevDecl)
-    : TagDecl(DK, TK, C, DC, IdLoc, Id, PrevDecl, StartLoc) {
+                       RecordDecl *PrevDecl, ArrayRef<TypedefDecl*> TypeParams,
+                       RecordDecl *BaseDecl,
+                       ArrayRef<TypeArgument> TypeArgs)
+    : TagDecl(DK, TK, C, DC, IdLoc, Id, PrevDecl, StartLoc),
+      IsGeneric(false), NumTypeParams(0), IsInstantiated(false), NumTypeArgs(0), IsDelayed(false) {
```

Comment:
IsGeneric(false), NumTypeParams(0), IsInstantiated(false), NumTypeArgs(0), IsDelayed(false): A better way would be to use member-wise initialization for all these data members when declaring them rather than do it at the constructor scope.

Reply:
Done